playhouse/postgres_ext.py

```diff
@@ -376,6 +401,80 @@ def _connect(self, database, **kwargs):
         return conn
 
 
+class PostgresqlExtUpdateQuery(UpdateQuery):
+    """Overrides peewee.UpdateQuery to add a returning feature"""
+
+    def __init__(self, model_class, update=None):
+        super(PostgresqlExtUpdateQuery, self).__init__(model_class, update)
+        self._tuples = False
+        self._dicts = False
+        self._naive = False
+        self._returning = False
+        self._qr = None
+
+    @returns_clone
+    def returning(self, *selection):
+        """Add a returning wrapper
+
+        Support for the PostgreSQL RETURNING keyword
+        """
+        self._returning = selection or self.model_class._meta.get_fields()
+
+        def naive(self, naive=True):
+            """Add a naive wrapper"""
+            self._naive = naive
+
+        def tuples(self, tuples=True):
+            """Add a tuples wrapper"""
+            self._tuples = tuples
+
+        def dicts(self, dicts=True):
+            """Add a dicts wrapper"""
+            self._dicts = dicts
+
+        setattr(PostgresqlExtUpdateQuery, 'naive', returns_clone(naive))
+        setattr(PostgresqlExtUpdateQuery, 'tuples', returns_clone(tuples))
+        setattr(PostgresqlExtUpdateQuery, 'dicts', returns_clone(dicts))
+
+    def _clone_attributes(self, query):
+        """Clone the specific attributes for chaining"""
+        query = super(PostgresqlExtUpdateQuery, self)._clone_attributes(query)
+        query._returning = self._returning
+        return query
+
+    def get_query_meta(self):
+        """Retrieve meta information of the current query"""
+        return (self._returning, self._joins)
+
+    def execute(self):
+        """Execute the current query
+
+        Depending on the use of returning or not a wrapped result is returned
+        """
+
+        if not self._returning:
+            return self.database.rows_affected(self._execute())
+
+        if self._dirty or not self._qr:
+            if self._tuples:
+                ResultWrapper = TuplesQueryResultWrapper
+            elif self._dicts:
+                ResultWrapper = DictQueryResultWrapper
+            elif self._naive:
+                ResultWrapper = NaiveQueryResultWrapper
+            else:
+                ResultWrapper = ModelQueryResultWrapper
```

Comment:
Since joins are not allowed on UPDATE queries with Peewee, I'm not sure of the case where you'd want the `ModelQueryResultWrapper` over the `NaiveQueryResultWrapper`.